test/functional/feature_tapscript_opcodes.py

```diff
@@ -6,10 +6,11 @@
 #
 # Test for taproot sighash algorithm with pegins and issuances
 
-from test_framework.key import compute_xonly_pubkey, generate_privkey, sign_schnorr, tweak_add_privkey, tweak_add_pubkey, verify_schnorr
```

Comment:
This test does not pass in the commit that it is added 14d5cac0acc486397c9330696f313405ff92cbbc

Reply:
also can you `chmod +x` the file?

Reply:
now added in 1dfe55190b40a33bd4d4217ad86e6b8c76621b34 .. but still does not pass